spec/example-unnamed-reifier.ttl

```diff
@@ -0,0 +1,6 @@
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
```

Comment:
I suggest using @prefix instead of PREFIX to make it consistent with other examples that are closed located.

Reply:
See above. (I should perhaps have done "modernizing" before this PR; but I want to land the hard stuff first.)